bindgen/templates/BytesHelper.go

```diff
@@ -0,0 +1,50 @@
+{#/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
`BytesHelper.go` doesn't have to be sequence template copy-pasta, `SequenceTemplate.go` can be reused.
[gen_go/mod.rs](https://github.com/NordSecurity/uniffi-bindgen-cs/blob/2b505e4d57aad440926f656245588fcbc07618f0/bindgen/src/gen_cs/mod.rs#L249C14-L250C1)
[Sequence template.](https://github.com/NordSecurity/uniffi-bindgen-cs/blob/2b505e4d57aad440926f656245588fcbc07618f0/bindgen/templates/Types.cs#L82)

Reply:
the main reason I did that, was I had hoped to be able to do some optimizations in there. but I guess that never happened

Reply:
Will you change this to reuse the sequence template, or do you still see possible optimizations here?